src/plcc/load_spec/validate_spec/validate_syntactic_spec/build_parsing_table_test.py

```diff
@@ -0,0 +1,90 @@
+from pytest import raises
+from .build_parsing_table import build_parsing_table, Table
+from collections import defaultdict
+
+def test_empty():
+    table = build_parsing_table({}, {}, {})
+    assert len(table.getFilledCellLocations()) == 0
+
+def test_set_cell():
```

Comment:
This should be deleted because the other tests cover this and it's not the point of the module. The point of the module is `build_parsing_table`, but this is testing the tools you use to get that done. Any test that tests `build_parsing_table` would implicitly be building the tools used as well.

Reply:
BTW, I think it's correct that you wrote this test to begin with. It's a good test to help you get off the ground in TDD. But then, once you have more relevant tests, these starter tests can go away.